### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by BahareMoradi**
refactoring

### CODE REVIEW

File: app/src/main/res/layout/activity_all_events.xml [hunk 1]
```diff
@@ -18,6 +19,7 @@
             android:id="@+id/event_back"
             android:layout_width="70dp"
             android:layout_height="70dp"
+            android:layout_weight="0.1"
```
Comment (Amirerfan) [inline app/src/main/res/layout/activity_all_events.xml:22]: I think that this code is not responsive.
please check it on tablet and large phones then comment here.
if you needed to commit any thing make sure you amend that and please change the commit text.

Comment (BahareMoradi) [reply app/src/main/res/layout/activity_all_events.xml:22]: well i ckecked it again and it works fine for me!
okay  i 'll keep that in mind!

Comment (Amirerfan) [reply app/src/main/res/layout/activity_all_events.xml:22]: Thank you!

**Review by Amirerfan — APPROVED** [review]
good job!

**Comment by alifarazz** [comment]
@BahareMoradi What did you refactor?

**Pull request merged**